src/html/template.html

```diff
@@ -1,4 +1,8 @@
 <div class="fl-mapviz-container">
+    <div id="fl-mapviz-disclaimer">
+        Please note that this map is for demonstration purposes only and may not accurately reflect the accessibiltiy or location of all COVID-19 test centres. To learn about the location of COVID-19 test centres, please visit your local health unit web site.
```

Comment:
```suggestion
        Please note that this map is for demonstration purposes only and may not accurately reflect the accessibility or location of all COVID-19 test centres. To learn about the location of COVID-19 test centres, please visit your local health unit web site.
```

Reply:
Good catch, @greatislander. I will issue a new PR for adding the disclaimer. This will be addressed there.